packet/ldap.py

```diff
@@ -79,6 +80,73 @@ def ldap_get_live_onfloor():
     return members
 
 
+def ldap_get_active_rtp():
```

Comment:
Looking back at these, this function and the other `ldap_get` functions I added should probably mostly be plural, since they're getting lists.